com.unity.multiplayer.mlapi/Runtime/NetworkedVar/INetworkedVar.cs

```diff
@@ -48,13 +48,17 @@ public interface INetworkedVar
         /// Reads the complete state from the reader and applies it
         /// </summary>
         /// <param name="stream">The stream to read the state from</param>
-        void ReadField(Stream stream);
+        /// <param name="localTick">The local network tick at which this var was written, on the machine it was written </param>
+        /// <param name="remoteTick">The remote network tick at which this var was sent by the host </param>
+        void ReadField(Stream stream, ushort localTick, ushort remoteTick);
         /// <summary>
         /// Reads delta from the reader and applies them to the internal value
         /// </summary>
         /// <param name="stream">The stream to read the delta from</param>
         /// <param name="keepDirtyDelta">Whether or not the delta should be kept as dirty or consumed</param>
-        void ReadDelta(Stream stream, bool keepDirtyDelta);
+        /// <param name="localTick">The local network tick at which this var was written, on the machine it was written </param>
+        /// <param name="remoteTick">The remote network tick at which this var was sent by the host </param>
+        void ReadDelta(Stream stream, bool keepDirtyDelta, ushort localTick, ushort remoteTick);
```

Comment:
if you think this PR introduces a breaking change, we should also call that out to be more cautious I think.